app/src/main/java/com/teamwss/websoso/ui/activityDetail/ActivityDetailViewModel.kt

```diff
@@ -3,34 +3,48 @@ package com.teamwss.websoso.ui.activityDetail
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
-import com.teamwss.websoso.data.repository.UserFeedRepository
+import androidx.lifecycle.viewModelScope
+import com.teamwss.websoso.data.repository.UserRepository
 import com.teamwss.websoso.ui.main.myPage.myActivity.model.ActivitiesModel.ActivityModel
+import com.teamwss.websoso.ui.mapper.toUi
 import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
 class ActivityDetailViewModel @Inject constructor(
-    private val activityRepository: UserFeedRepository,
+    private val userRepository: UserRepository,
 ) : ViewModel() {
 
     private val _userActivity = MutableLiveData<List<ActivityModel>>()
     val userActivity: LiveData<List<ActivityModel>> get() = _userActivity
 
-    init {
-//        updateMyActivities()
+    private val _userId: MutableLiveData<Long> = MutableLiveData()
+    val userId: LiveData<Long> get() = _userId
+
+    private val _lastFeedId = MutableLiveData<Long>().apply { value = 0L }
```

Comment:
```suggestion
    private val _lastFeedId =  MutableLiveData<Long>(0L)
```
r: 이렇게 가능한걸로 알고 있습니다.

Reply:
r: 예전에도 리뷰 달았지만 livedata 초기화에 apply를 쓰지않아요

Reply:
아앗.. 수정하겠습니다!